tests/test_parted_disk.py

```diff
@@ -88,11 +90,20 @@ def runTest(self):
         # TODO
         self.fail("Unimplemented test case.")
 
-@unittest.skip("Unimplemented test case.")
-class DiskAddPartitionTestCase(unittest.TestCase):
+class DiskAddPartitionTestCase(RequiresDisk):
+    """addPartition should return True if partition is added successfully(even
```

Comment:
See previous comment by @bcl about these starting/trailing triple quotes. Need to be on a line by itself.